SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialTest1x2GridSmallInterestWorkerSettings.cpp

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialTest1x2GridSmallInterestWorkerSettings.h"
+
+#include "SpatialTest1x2GridSmallInterestStrategy.h"
+
+USpatialTest1x2GridSmallInterestWorkerSettings::USpatialTest1x2GridSmallInterestWorkerSettings()
+{
+	WorkerLayers[0].LoadBalanceStrategy = USpatialTest1x2GridSmallInterestStrategy::StaticClass();
```

Comment:
worth check-ing whether we have an array entry?

Reply:
Hm, I think we use the same elsewhere as well, relying on the fact that the worker layers have the first layer already.
Since this is used in premerge tests, we will catch it if the implementation changes, so I think it's okay.